### PR TITLE
Refactor pagination

### LINKED ISSUE #222: In pagination links don't use sort parameter when default order is used

### PR DESCRIPTION
**Pull request by vjik**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ❌
| Breaks BC?    | ✔️
Fix #222

### CODE REVIEW

**Review by samdark — CHANGES_REQUESTED** [review]
Overall :+1: but some things to be addressed.

File: config/widgets-themes.php [hunk 1]
```diff
@@ -66,7 +66,6 @@
             'itemTag()' => ['li'],
             'itemAttributes()' => [['class' => 'page-item']],
             'linkAttributes()' => [['class' => 'page-link']],
-            'currentItemClass()' => ['active'],
```
[the remark this answers is not in the document]
Comment (vjik) [reply config/widgets-themes.php:69]: There isn't this method in `KeysetPagination`.

File: src/BaseListView.php [hunk 2]
```diff
@@ -126,7 +127,7 @@ abstract class BaseListView extends Widget
 
     private array $offsetPaginationConfig = [];
     private array $keysetPaginationConfig = [];
-    private string|OffsetPagination|KeysetPagination|null $pagination = null;
+    private PaginationControlInterface|null $paginationControl = null;
```
[the remark this answers is not in the document]
Comment (vjik) [reply src/BaseListView.php:130]: I call it "control" by analogy with "PageSizeControlInterface". In forms such elements are called "control" also.

May be "widget" is better for pagination and page size also. WDYT?

[the remark this answers is not in the document]
Comment (samdark) [reply src/BaseListView.php:130]: Yes. I think that widget is better.

[the remark this answers is not in the document]
Comment (vjik) [reply src/BaseListView.php]: done

**Review by samdark — APPROVED** [review]

**Pull request merged**